Watchman.Discord/Areas/Protection/Services/UnmutingService.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Devscord.DiscordFramework.Framework.Commands.Responses;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Devscord.DiscordFramework.Services;
+using Devscord.DiscordFramework.Services.Factories;
+using Devscord.DiscordFramework.Services.Models;
+using Serilog;
+using Watchman.DomainModel.Users;
+
+namespace Watchman.Discord.Areas.Protection.Services
+{
+    public class UnmutingService : ICyclicService
+    {
+        public RefreshFrequent RefreshFrequent { get; } = RefreshFrequent.Quarterly;
```

Comment:
czemu tutaj jest ustawione na sztywno?

Reply:
z założenia ICyclicService ma mieć czas odświeżania na sztywno

Reply:
myśle że fajnie gdyby to było konfigurowalne z góry
żeby był taki jeden punkt główny, który by to wszystko ustawiał